### DISCUSSION TITLE
Cannot start the macOS executable.

### DISCUSSION THREAD

**Discussion by Otooooo** [KevinBytesTheDust/CVAmp#203] [Q&A]
<img width="531" alt="Screenshot 2023-09-27 at 17 16 42" src="https://github.com/jlplenio/crude-twitch-viewer-bot/assets/115180163/a65711b8-90c7-41b8-905d-17e153bcd7b2">
this file is a .bin and this happen when i open it
<img width="384" alt="Screenshot 2023-09-27 at 17 17 01" src="https://github.com/jlplenio/crude-twitch-viewer-bot/assets/115180163/d75f33fa-9fe2-45ba-99b3-e7f87b852c6f">

**Comment by jlplenio** [comment]
Hello you. You need to launch the .bin from the command line.

But I just built a new macOS app bundle that you should double-click to launch.
I cannot test it - can you try this and let me know? https://we.tl/t-dMoAwFkE8w

**Reply by okayblast** [reply]
Can you re-upload that file please? I'm also having issues with opening the file on my Macbook. 
<img width="373" alt="image" src="https://github.com/jlplenio/crude-twitch-viewer-bot/assets/159509411/37e9e860-4115-4172-b946-7058dd864b4c">

**Reply by jlplenio** [reply]
@Otooooo @okayblast macOS could and should work in the newest pre-release. Please try:
https://github.com/jlplenio/crude-twitch-viewer-bot/releases/tag/v0.6.2
